scss/bootstrap/overrides/_variables.scss

```diff
@@ -326,67 +297,68 @@ $crm-nav-tabs-panel-heading-link-weight:             600 !default;
 //
 //##
 
-$pagination-color:                     $gray-dark !default;
-$pagination-bg:                        transparent !default;
+$pagination-color:               $gray-dark !default;
+$pagination-bg:                  transparent !default;
 
-$pagination-hover-color:               $gray-darker !default;
-$pagination-hover-bg:                  $pagination-bg !default;
+$pagination-hover-color:         $gray-darker !default;
+$pagination-hover-bg:            $pagination-bg !default;
 
-$pagination-active-color:              $gray-darker !default;
-$pagination-active-bg:                 $pagination-bg !default;
+$pagination-active-color:        $gray-darker !default;
+$pagination-active-bg:           $pagination-bg !default;
 
-$pagination-disabled-color:            $gray-light !default;
-$pagination-disabled-bg:               $pagination-bg !default;
+$pagination-disabled-color:      $gray-light !default;
+$pagination-disabled-bg:         $pagination-bg !default;
 
 //== Form states and alerts
 //
 //## Define colors for form feedback states and, by default, alerts.
 
-$state-success-text:             $brand-success !default;
+$state-success-text:             #4D994D !default;
 $state-info-text:                $brand-info !default;
-$state-warning-text:             $brand-warning !default;
+$state-warning-text:             #BF5900 !default;
```

Comment:
where is this color coming from? is it part of the palette? if no, it should be removed. if yes, it should be declared first of all as a primary color, then assigned to this variable

Reply:
same as above

Reply:
see above